.gitignore

```diff
@@ -2,3 +2,5 @@ vendor/*
 report/*
 phpunit.xml
 composer.lock
+
+.idea/
```

Comment:
Yeah cool, This wasn't meant to be part if the pull request - I don't have a lot of experience with pull requests.